CONTRIBUTING.md

```diff
@@ -47,10 +47,10 @@ Chart releases must be immutable. Any change to a chart warrants a chart version
 
 The chart `version` should follow [semver](https://semver.org/).
 
-Charts should start at `1.0.0`. Any breaking (backwards incompatible) changes to a chart should:
+Charts should start at `1.0.0`. All changes should be described in the [Chart metadata](https://docs.k8s-at-home.com/our-helm-charts/development/creating-a-new-chart/#chart-metadata). Additionally, any breaking (backwards incompatible) changes to a chart should: 
 
 1. Bump the MAJOR version
-2. In the README, under a section called "Upgrading", describe the manual steps necessary to upgrade to the new (specified) MAJOR version
+1. Include `**BREAKING**` in the change description and describe the manual steps necessary to upgrade to the new (specified) MAJOR version
```

Comment:
I know some of this wording was already in place, but maybe we should change it a bit. I am not sure we really want extensive upgrade instructions, but I do believe that any breaking changes should be  communicated. I'm not sure what the correct balance would be though 😅

Reply:
Yes, I think the main motivation is just to remove what is here which I think is incorrect. So having less detail makes sense as it would be less to maintain.

How about this?  I've tried to leave the spirit of the message here (to tell people that major versions are important, and it is important to describe upgrade steps) but still linked to the more detailed chart instructions that describe the annotation metadata.